docs/index.md

```diff
@@ -0,0 +1,23 @@
+# Welcome to Splunk Connect for Syslog
+
+Splunk Connect for Syslog is an open source packaged solution for 
+getting data into Splunk using syslog-ng (OSE) and the Splunk 
+HTTP event Collector. 
+
+## Project Goals
+
+* Bring a tested configuration and build of syslog-ng OSE to the market that will function consistently regardless of linux distribution
```

Comment:
Can the solution really support any type of Linux? Even if true I think it might better to state this more conservatively. "validated to function consistently on RedHat Base Image, etc, etc." 
So the scope is defined.

Reply:
our regardless is limited to the set of distros supported by docker so we are fairly safe. That is actually what we mean their host can be Suse or RHEL we can give the syslog as an app container no matter what their standard base is